### ISSUE TITLE
Topology not displayed with NetBox 3.3.9

### ISSUE THREAD

**Issue by thford89** [mattieserver/netbox-topology-views#194]
There appears to be an issue with the plugin with NetBox 3.3.9 in docker with a fresh install. 

I have checked the filters but cant seem to get anything to display. 

![image](https://user-images.githubusercontent.com/10763519/206060824-e45de43f-2e6e-4f4e-8769-53b5d3c28f98.png)

![image](https://user-images.githubusercontent.com/10763519/206061022-dd86bbbe-dbc5-4f27-a753-99ea0036ddf0.png)

**Comment by dreng** [comment]
This seems to be a general issue as I just updated my test instance to NetBox 3.3.9 and (as a requirement) Topology Views 3.1.0 and ran into the same problem.

After setting debug to true I got the following error:

> SuspiciousFileOperation at /plugins/netbox_topology_views/topology/
>
> The joined path (/netbox_topology_views/img/role-unknown.png) is located outside of the base path component (/opt/netbox/netbox/project-static/dist)

Gonna dive into it a little more...

**Comment by dreng** [comment]
I have to correct myself: The error only appears with debugging turned on and seems to be related to the debugging feature itself.

**Comment by dreng** [comment]
This issue is not related to NetBox 3.3.9. I just reverted to 3.3.8 and got the same result.

A current workaround is to downgrade to Topology Views 3.0.1 until a fix has been released. This can be achieved by editing /opt/netbox/local_requirements.txt.

- cd /opt/netbox
- Edit local_requirements.txt: Change "netbox_topology_views" to "netbox_topology_views==3.0.1"
- Run "./upgrade.sh"
- Restart NetBox service

**Comment by thford89** [comment]
@dreng I switched to 3.0.1 but seem to be getting the same result.

**Comment by dreng** [comment]
@thford89
Okay, this is very confusing. I played around a bit with the installations and now the current plugin works with the current NetBox version. However, nothing has been changed in total. It could be that there was a caching problem. Try opening NetBox with a cleared cache or in another browser or a private session. I would also restart the netbox and netbox-rq services first, just to be sure.

I don't use Docker and therefore don't know if the whole thing behaves differently there.
